### PR TITLE
Update simplefm_example.php

### PR DESCRIPTION
**Pull request by jeremiahsmall**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -26,11 +26,14 @@
     },
     "require-dev": {
         "php-http/curl-client": "^1.6",
```
Comment (DASPRiD) [inline composer.json]: Actually, why do we have an HTTP client implementation and a log implementation in require-dev? That doesn't make much sense.

File: composer.json [hunk 2]
```diff
@@ -26,11 +26,14 @@
...
+        "monolog/monolog": "^1.9",
         "zendframework/zend-crypt": "^3.1",
         "phpunit/PHPUnit": "^5.5",
         "squizlabs/php_codesniffer": "^2.6"
     },
     "suggest": {
+        "php-http/curl-client": "For creating Connections",
```
Comment (DASPRiD) [inline composer.json]: Same here, suggest is not meant for this kind of stuff.

File: doc/filemaker/simplefm_example.php [hunk 3]
```diff
@@ -9,115 +9,153 @@
...
-$errorType,
-$count,
-$fetchSize
+    $uri,
+    $database,
+    (string) $command,
+    $connectionLogPath,
+    $errorLogPath,
+    $records->getTotalCount(),
+    $records->count()
```
Comment (DASPRiD) [inline doc/filemaker/simplefm_example.php]: Since we have a countable interface, do count($records)

File: doc/filemaker/simplefm_example.php [hunk 4]
```diff
@@ -127,28 +165,39 @@ function stringifyValue($value) : string
...
+                $tempValue .= PHP_EOL;
             }
-            $value = $tempValue;
+            $collapsedValue .= $tempValue;
         } else {
             // repeating field
-            $value = implode("<br>", $value);
+            $collapsedValue .= implode(PHP_EOL, $value);
         }
-    } elseif ($value instanceof DateTimeImmutable) {
```
Comment (DASPRiD) [inline doc/filemaker/simplefm_example.php]: Check for interface `DateTimeInterface` instead.

File: doc/filemaker/simplefm_example.php (hunk 3 above)
Comment (jeremiahsmall) [inline doc/filemaker/simplefm_example.php]: Fixed

File: composer.json (hunk 2 above)
Comment (jeremiahsmall) [inline composer.json]: Removed opinionated suggests.

File: composer.json (hunk 1 above)
Comment (jeremiahsmall) [inline composer.json]: I've moved them into `doc/filemaker/composer.json`. Much nicer like this .

**Pull request merged**